app/bundles/CoreBundle/Guzzle/ClientFactoryInterface.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Mautic\EmailBundle\Swiftmailer\Guzzle;
+namespace Mautic\CoreBundle\Guzzle;
```

Comment:
@escopecz I am moving Guzzle to the CoreBundle, unless we want to use another client, then it is better to remove it.

Reply:
I wonder why there is a need to have a special Guzzle services in Mautic code at all. Is that used anywhere? I don't see any usage.

Reply:
I do not think it is being anywhere else, @fedys let's consider removing these files as we do not need them anymore

Reply:
I'm for removing it if it is not used anywhere.